src/frequenz/sdk/timeseries/battery_pool/_methods.py

```diff
@@ -183,7 +183,7 @@ def _remove_metric_fetcher(
         _logger.error(
             "Removing component %d from the %s formula.",
             component_id,
-            self._result_channel.name,
+            self._result_channel._name,  # pylint: disable=protected-access
```

Comment:
BTW, we are adding the name as a public (but now read-only) attributed back on the next release :grimacing: